homeassistant/components/litterrobot/select.py

```diff
@@ -19,14 +19,24 @@
 from .entity import LitterRobotEntity, _RobotT
 from .hub import LitterRobotHub
 
-_CastTypeT = TypeVar("_CastTypeT", int, float)
+_CastTypeT = TypeVar("_CastTypeT", int, float, str)
+
+BRIGHTNESS_LEVEL_MAPPER: dict[BrightnessLevel | None, tuple[str | None, str]] = {
+    BrightnessLevel.LOW: ("Low", "mdi:lightbulb-on-30"),
+    BrightnessLevel.MEDIUM: ("Medium", "mdi:lightbulb-on-50"),
+    BrightnessLevel.HIGH: ("High", "mdi:lightbulb-on"),
```

Comment:
Please translate the select option instead of using English: https://developers.home-assistant.io/docs/internationalization/core#selectors

Reply:
Is there a recommendation for enums -> state/options/translation_key, specifically around capitalization? For example, if I just call the `.name` property for the enum, I'd get `LOW`. If I used that value in the state/options, it's easy (and less code) to cast it back with `BrightnessLevel["LOW"]` and have a translation that displays as `Low` rather than having to use `.lower()` to store the state and `.upper()` to lookup or, alternatively, creating a map for it.